android/app/buildConfigs.gradle

```diff
@@ -13,10 +17,11 @@ def createBuildConfig(buildConfigName) {
     logger.quiet("Using build config $buildConfigName")
 
     def command
-    if (System.properties['os.name'].toLowerCase().contains('windows')) {
-        command = ["cmd", "/c", "yarn --silent babel-node tools/build-config to-json $buildConfigName"]
+    def baseCommand = "yarn --silent babel-node tools/build-config to-json $buildConfigName"
+    if (isWindows() == "true") {
+        command = ["cmd", "/c", baseCommand]
     } else {
-        command = "yarn --silent babel-node tools/build-config to-json $buildConfigName"
+        command = baseCommand
     }
     def proc = command.execute()
     proc.waitFor()
```

Comment:
This (and 4 more lines) is exactly the same as below, right? Could you maybe move it to a separate function?